namelayer-spigot/src/main/java/vg/civcraft/mc/namelayer/database/GroupManagerDao.java

```diff
@@ -1004,21 +844,43 @@ public void run() {
 	}
 	
 	public void removeMember(UUID member, Group group){
-		//TODO Redo this
+		//TODO Redo remove member
 		try (Connection connection = db.getConnection();
-				PreparedStatement removeMember = connection.prepareStatement(this.removeMember)){
+				PreparedStatement removeMember = connection.prepareStatement(GroupManagerDao.removeMember)){
 			removeMember.setString(1, member.toString());
 			//removeMember.setString(2, group);
 			removeMember.executeUpdate();
 		} catch (SQLException e) {
 			logger.log(Level.WARNING, "Problem removing " + member + " from group " + group, e);
 		}
 	}
+
+	public void removeAllMembersAsync(final Group group){
```

Comment:
But while the sync part is at least used in that old db upgrade, the async one literally isn't used at all? The only time where we would want to delete all members is when  deleting a group and that is being done in the stored procedure.